src/main/java/uk/nhs/tis/trainee/notifications/migration/DeleteUnusedPmUpdateHistory.java

```diff
@@ -38,7 +38,7 @@
  * Delete unused programme-membership-update history records.
  */
 @Slf4j
-@ChangeUnit(id = "deleteUnusedPmUpdateHistory", order = "2", runAlways = true)
+@ChangeUnit(id = "deleteUnusedPmUpdateHistory", order = "2")
```

Comment:
Turn off the ongoing clearing of the history in anticipation of actually sending mails